cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -88,6 +88,30 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
             }
           });
         });
+
+        // See https://github.com/bbc/simorgh/pull/4352 for more information.
+        if (service === 'news') {
+          it('plays media when a user clicks play', () => {
+            cy.window().then(win => {
+              const media = getBlockData('video', win.SIMORGH_DATA.pageData);
+              if (media && media.type === 'video') {
+                cy.get(
+                  'div[class^="StyledVideoContainer"] img[class^="StyledImg"]',
```

Comment:
We should (later) change this to use a more reliable selector - there is a [core pod issue to look at this problem](
https://github.com/bbc/simorgh/issues/3714).

Reply:
Yes, I agree. I think I wrote an issue that might now be a duplicate of that? https://github.com/bbc/simorgh/issues/4297

Reply:
Yeah I think close yours in favour of the other one that has more comments.